### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Remove usage of travis file, replace by actions
This could be turned into a template (TBD). Those repositories are not changed very often.
I will look at the other repositories afterwards if we are happy

* Release creation tested: https://github.com/jburel/zeroc-ice-ubuntu2004/releases/tag/0.2.1
* General build https://github.com/jburel/zeroc-ice-ubuntu2004/actions/runs/387080072

cc @sbesson @joshmoore

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Create a release
+        if: startsWith(github.ref, 'refs/tags')
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+      - name: Upload binaries to release
+        uses: svenstaro/upload-release-action@v2
```
Comment (sbesson) [inline .github/workflows/main.yml:40]: Is the wildcard the primary  motivation for using this rather than https://github.com/actions/upload-release-asset?

**Review by sbesson — COMMENTED** [review]
Overall looks good. Only question is related to using the core release asset action vs a custom action which I assume is related to the question around the workflow template. How many repos would benefit from a similar workflow?

File: .github/workflows/main.yml (hunk 1 above)
Comment (jburel) [reply .github/workflows/main.yml:40]: yes since we might have different types of artifacts to push.

Comment (jburel) [reply .github/workflows/main.yml:40]: We have few repositories following the same patterns. (debian, centos, ubuntu) we could do with having a template.

Comment (jburel) [reply .github/workflows/main.yml:40]: https://github.com/actions/upload-release-asset/issues/47

**Comment by jburel** [comment]
Last commit tested via https://github.com/jburel/zeroc-ice-ubuntu2004/releases/tag/0.2.2

**Comment by jburel** [comment]
I reviewed the logic following today's discussion 
https://github.com/jburel/zeroc-ice-ubuntu2004/releases/tag/0.2.3
I think I will turn it into a template so it can be used in other repositories

**Comment by jburel** [comment]
I need to push a commit for badge

**Review by manics — APPROVED** [review]
LGTM

**Pull request merged**